Samiksha/html/Assignment2/Assign1.html

```diff
@@ -35,12 +39,7 @@ <h1> Initial public offering </h1>
   At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del> <ins>20</ins><br>
   years, until the year 2024.
 </p>
-<center style="color:red"> <h2>Thank You!!</h2> </center>
+<h2>Thank You!!</h2>
  </body>
```

Comment:
indentation is not proper there should be always four tab space between parent and children tag.